src/connectors/github_organization.py

```diff
@@ -0,0 +1,339 @@
+    ('raw', 'VARIANT'),
+    ('hash_raw', 'NUMBER'),
+    ('ref', 'VARCHAR(256)'),
+    ('before', 'VARCHAR(256)'),
+    ('after', 'VARCHAR(256)'),
+    ('created', 'BOOLEAN'),
+    ('deleted', 'BOOLEAN'),
+    ('forced', 'BOOLEAN'),
+    ('base_ref', 'VARCHAR(256)'),
+    ('compare', 'VARCHAR(256)'),
+    ('commits', 'VARIANT'),
+    ('head_commit', 'VARIANT'),
+    ('repository', 'VARIANT'),
+    ('pusher', 'VARIANT'),
+    ('organization', 'VARIANT'),
+    ('sender', 'VARIANT'),
+    ('action', 'VARCHAR(256)'),
+    ('check_run', 'VARIANT'),
+    ('check_suite', 'VARIANT'),
+    ('number', 'NUMBER(38,0)'),
+    ('pull_request', 'VARIANT'),
+    ('label', 'VARIANT'),
+    ('requested_team', 'VARIANT'),
+    ('ref_type', 'VARCHAR(256)'),
+    ('master_branch', 'VARCHAR(256)'),
+    ('description', 'VARCHAR(256)'),
+    ('pusher_type', 'VARCHAR(256)'),
+    ('review', 'VARIANT'),
+    ('changes', 'VARIANT'),
+    ('comment', 'VARIANT'),
+    ('issue', 'VARIANT'),
+    ('id', 'NUMBER(38,0)'),
+    ('sha', 'VARCHAR(256)'),
+    ('name', 'VARCHAR(256)'),
+    ('target_url', 'VARCHAR(8192)'),
+    ('context', 'VARCHAR(256)'),
+    ('state', 'VARCHAR(256)'),
+    ('commit', 'VARIANT'),
+    ('branches', 'VARIANT'),
+    ('created_at', 'TIMESTAMP_LTZ(9)'),
+    ('updated_at', 'TIMESTAMP_LTZ(9)'),
+    ('assignee', 'VARIANT'),
+    ('release', 'VARIANT'),
+    ('membership', 'VARIANT'),
+    ('alert', 'VARIANT'),
+    ('scope', 'VARCHAR(256)'),
+    ('member', 'VARIANT'),
+    ('requested_reviewer', 'VARIANT'),
+    ('team', 'VARIANT'),
+    ('starred_at', 'TIMESTAMP_LTZ(9)'),
+    ('pages', 'VARIANT'),
+    ('project_card', 'VARIANT'),
+    ('build', 'VARIANT'),
+    ('deployment_status', 'VARIANT'),
+    ('deployment', 'VARIANT'),
+    ('forkee', 'VARIANT'),
+    ('milestone', 'VARIANT'),
+    ('key', 'VARIANT'),
+    ('project_column', 'VARIANT'),
+    ('status', 'VARCHAR(256)'),
+    ('avatar_url', 'VARCHAR(256)')
+]
+
+CONNECT_RESPONSE_MESSAGE = """
+STEP 1: Modify the Role "{role}" to include the following trust relationship:
+
+{role_trust_relationship}
+
+
+STEP 2: For Role "{role}", add the following inline policy:
+
+{role_policy}
+"""
+
+
+def connect(connection_name, options):
+    base_name = f'GITHUB_ORGANIZATION_{connection_name}_EVENTS'.upper()
+    stage = f'data.{base_name}_STAGE'
+    staging_table = f'data.{base_name}_STAGING'
+    landing_table = f'data.{base_name}_CONNECTION'
+
+    bucket = options['bucket_name']
+    prefix = options.get('filter', S3_BUCKET_DEFAULT_PREFIX)
+    role = options['aws_role']
+
+    comment = f"""
+---
+module: github_organization
+"""
+
+    db.create_stage(
+        name=stage,
+        url=f's3://{bucket}',
+        prefix=prefix,
+        cloud='aws',
+        credentials=role,
+        file_format=FILE_FORMAT
+    )
+
+    db.create_table(
+        name=staging_table,
+        cols=[
+            ('value', 'VARIANT'),
+            ('filename', 'STRING(200)')
+        ]
+    )
```

Comment:
I don't think you need the staging table/pipe/task model of connector here. That structure gets used for Cloudtrail because cloudtrail logs need to be flattened before they can be usefully queried. It doesn't look like the github logs need to be flattened (or at least your ingest task doesn't flatten anything when it pulls them from the stream). 

This means you can simplify the connector by modifying the pipe to copy files from S3 into the landing table and then you don't need the stream, task, and extra table.